project/bin/main/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.application.name=project
+
+spring.datasource.url=jdbc:mysql://localhost:3307/securityDB
```

Comment:
This will break when merged, as datasource is now ``spring.datasource.url=jdbc:mysql://db:3306/securityDB``

Reply:
I think I committed this by mistake, I will just copy the project application.properties from the dev branch and commit that and then create a new pull request to the dev branch